src/pages/home/report/comment/TextCommentFragment.tsx

```diff
@@ -69,7 +74,10 @@ function TextCommentFragment({fragment, styleAsDeleted, styleAsMuted = false, so
         if (containsOnlyEmojis) {
             htmlContent = Str.replaceAll(htmlContent, '<emoji>', '<emoji islarge>');
             htmlContent = Str.replaceAll(htmlContent, '<blockquote>', '<blockquote isemojisonly>');
+        } else if (CONST.REGEX.ALL_EMOJIS.test(text ?? '')) {
```

Comment:
What is the difference between `containsOnlyEmojis` and `ALL_EMOJIS`?

Reply:
I've put `CONST.REGEX.ALL_EMOJIS.test(text ?? '')` into `containsEmojis` const to make it clear : https://github.com/Expensify/App/pull/47547/commits/921b364338526975fcddc5dd7e71f9b5bbf7fbc2

`containsOnlyEmojis` => will be true for cases with emojis only `😄 😸 😃`
while `containsEmojis` also will be true for cases like `hello 😄`